### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by ciarams87**
### Proposed changes
Documentation and bundle updates for the 0.3.0 release

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-ingress-operator/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: examples/deployment-oss-min/nginx-ingress-controller.yaml [hunk 1]
```diff
@@ -7,9 +7,9 @@ spec:
   type: deployment
   nginxPlus: false
   image:
-    repository: registry.hub.docker.com/nginx/nginx-ingress
-    tag: edge-ubi
+    repository: nginx/nginx-ingress
+    tag: 1.12.0-ubi
```
Comment (vepatel) [inline examples/deployment-oss-min/nginx-ingress-controller.yaml:11]: Minor one: should we do the same in `deployment-plus-min` as well? considering we now specify the latest release tag in KIC master and release but in operator it remains the same i.e. `edge` in both master and release.

**Review by vepatel — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -92,6 +92,8 @@ vet: ## Run go vet against code.
 lint: ## Run golangci-lint against code.
 	docker run --pull always --rm -v $(shell pwd):/nginx-ingress-operator -w /nginx-ingress-operator -v $(shell go env GOCACHE):/cache/go -e GOCACHE=/cache/go -e GOLANGCI_LINT_CACHE=/cache/go -v $(shell go env GOPATH)/pkg:/go/pkg golangci/golangci-lint:latest golangci-lint --color always run
 
+unit-test:
+	GO111MODULE=on go test ./... -coverprofile cover.out
```
Comment (lucacome) [inline Makefile:96]:
```suggestion
	go test ./... -coverprofile cover.out
```

no need for the flag

File: README.md [hunk 3]
```diff
@@ -41,28 +46,25 @@ It is possible to run the operator in your local machine. This is useful for tes
...
-The operator will run in your local machine but will be communicating with the cluster. The operator will only watch the `default` namespace when deployed locally.
+The operator will run in your local machine but will be communicating with the cluster. 
 
 ### Update CRD
 
 If any change is made in the CRD in the go code, run the following commands to update the changes in the CRD yaml:
 
-1. `make generate-crds`
-1. Apply the new CRD definition again in your cluster `kubectl apply -f deploy/crds/k8s.nginx.org_nginxingresscontrollers_crd.yaml`.
+1. `make generate`
```
Comment (lucacome) [inline README.md:62]:
```suggestion
1. `make manifests`
```

File: bundle/manifests/k8s.nginx.org_nginxingresscontrollers.yaml [hunk 4]
```diff
@@ -196,14 +196,12 @@ spec:
...
-                      of the Prometheus endpoint. If the argument is set,
-                      but the Ingress controller is not able to fetch the Secret from
-                      Kubernetes API, the Ingress Controller will fail to start. Format
-                      is namespace/name.
+                    description: Specifies an TLS Secret in the format namespace/name
+                      to use to secure the Prometheus endpoint.
                     type: string
                 required:
                 - enable
+                - secret
```
Comment (lucacome) [inline bundle/manifests/k8s.nginx.org_nginxingresscontrollers.yaml:204]: Not sure if this change should be part of the Release PR.

**Review by lucacome — CHANGES_REQUESTED** [review]

File: bundle/manifests/k8s.nginx.org_nginxingresscontrollers.yaml (hunk 4 above)
Comment (pleshakov) [reply bundle/manifests/k8s.nginx.org_nginxingresscontrollers.yaml:204]: I sent a PR https://github.com/nginxinc/nginx-ingress-operator/pull/133

Comment (ciarams87) [reply bundle/manifests/k8s.nginx.org_nginxingresscontrollers.yaml:204]: It came in when I generated the bundle - I can generate it separately.

**Review by soneillf5 — APPROVED** [review]

**Pull request merged**